src/cascadia/TerminalApp/SuggestionsControl.cpp

```diff
@@ -722,6 +722,17 @@ namespace winrt::TerminalApp::implementation
                     // ever allow non-sendInput actions.
                     DispatchCommandRequested.raise(*this, actionPaletteItem.Command());
 
+                    if (til::starts_with(actionPaletteItem.Command().ActionAndArgs().Args().as<SendInputArgs>().Input(), L"winget"))
```

Comment:
Do we need nullptr or `try_as` checks here?
`actionPaletteItem.Command()` is also called above. Should we assign it to a variable?

Reply:
They're all `SendInput` actions rn, but I'll make the change to future-proof it.